WallpaperChanger/wallpaper.py

```diff
@@ -98,15 +75,53 @@ def loadImage(self, img_path):
         # First check that file exists.
         if self._image_exists(img_path):
             try:
-                print("{} loaded.".format(img_path))
+                self.vprint("'{}' loaded.".format(img_path))
                 ctypes.windll.user32.SystemParametersInfoW(
                     SPI_SETDESKWALLPAPER, 0, img_path, 3)
             except Exception:
                 print(traceback.format_exc())
         else:
-            print("{} does not exist!".format(img_path))
+            self.vprint("{} does not exist!".format(img_path), level=logging.WARNING)
+
+    def vprint(self, message, level=logging.INFO):
+        """Prints out the message only if program is run in verbose mode.
 
-    def vprint(self, message):
-        """Prints out the message only if program is run in verbose mode."""
+        Always prints to log.
+        """
         if self.verbose:
-            print(message)
\ No newline at end of file
+            print(message)
+
+        logging.log(level=level, msg="{}".format(message))
```

Comment:
`logging.log(level=level, msg="{}".format(message))` is redundant. 

Can use `logging.log(level=level, msg=message)` instead.